app/models/time_profile.rb

```diff
@@ -71,7 +71,7 @@ def tz_or_default(default_tz = DEFAULT_TZ)
   end
 
   def days
-    profile[:days]
+    profile["days"] || profile[:days]
```

Comment:
Where is this being set?  It'd probably be better to set this consistently rather than check both string and symbol keys everywhere this is referenced.

Reply:
This was an issue that we ran into after creating time profiles using the react time profile form. Since that form has not been merged yet this pr does can be kept as just a WIP and fix this issue once the form has been merged. For reference: https://github.com/ManageIQ/manageiq-ui-classic/pull/7761 this is the time profile form conversion pr. The time profile day and hour data are posted as strings so that causes the error on the templates page.

Reply:
Why not fix the form then to save the profile data as string keys so that this isn't needed at all then is my point

Reply:
I could do that but after we discussed what to do with the time profiles form we decided to keep it as strings and instead start this issue https://github.com/ManageIQ/manageiq/issues/21294

Reply:
Yeah there are a bunch of good suggestions there, just checking both on access feels like the most fragile though